src/ResourceManager/KeyVault/Commands.KeyVault/help/Remove-AzureKeyVaultManagedStorageAccount.md

```diff
@@ -83,6 +95,19 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -InRemovedState
+Permanently remove the previously deleted managed storage account.```yaml
```

Comment:
Move this onto a new line.

Reply:
Thanks, fixed. Not sure why platyPs generates this on the same line.